tests/Unit/InvoiceTest.php

```diff
@@ -307,4 +307,66 @@ public function test_it_can_determine_if_reverse_charge_applies()
 
         $this->assertTrue($invoice->reverseChargeApplies());
     }
+    public function test_it_can_determine_if_is_deleted(): void
+    {
+        $customerId = 'foo';
+
+        $stripeInvoice = new StripeInvoice();
+
+        $stripeInvoice->customer = $customerId;
+        $stripeInvoice->status = StripeInvoice::STATUS_DELETED;
```

Comment:
Note that `Stripe\Invoice::STATUS_DELETED` has been deprecated since [`stripe/stripe-php:10.12.0`](https://github.com/stripe/stripe-php/releases/tag/v10.12.0) and has been removed with [`stripe/stripe-php:^12.0.0`](https://github.com/stripe/stripe-php/blob/v12.0.0/CHANGELOG.md#other-changes).